mimic/model/flavor_objects.py

```diff
@@ -0,0 +1,76 @@
+"""
+Model objects for mimic flavors.
+"""
+
+from characteristic import attributes
+
+
+@attributes(['flavor_id', 'name', 'ram', 'tenant_id'])
+class Flavor(object):
+    """
+    A Flavor object
+    """
+
+    static_defaults = {
+        "vcpus": 20,
+        "swap": "",
+        "rxtx_factor": 10000.0,
+        "OS-FLV-EXT-DATA:ephemeral": 0,
+        "disk": 800
+    }
+
+    def links_json(self, absolutize_url):
+        """
+        Create a JSON-serializable data structure describing the links to this
+        flavor.
+        """
+        return [
+            {
+                "href": absolutize_url("v2/{0}/flavors/{1}"
+                                       .format(self.tenant_id, self.flavor_id)),
+                "rel": "self"
+            },
+            {
+                "href": absolutize_url("{0}/flavors/{1}"
+                                       .format(self.tenant_id, self.flavor_id)),
+                "rel": "bookmark"
+            }
+        ]
+
+    def extra_specs_json(self):
+        """
+        Create a JSON-serializable data structure describing
+        ``OS-FLV-WITH-EXT-SPECS:extra_specs`` for a flavor.
+        """
+        return {
+            "quota_resources": "instances=mimic-instances,ram=mimic-ram",
+            "class": "mimic",
```

Comment:
@jimrollenhagen: Also, Reach is using this as well and I was afraid that they might be needing all the Rackspace specific info.